src/Nest/_Generated/_Requests.generated.cs

```diff
@@ -6537,8 +6563,6 @@ public partial class ValidateQueryRequest<T> : PlainRequestBase<ValidateQueryReq
 		public bool? AllowNoIndices { get => Q<bool?>("allow_no_indices"); set => Q("allow_no_indices", value); }
 		///<summary>Whether to expand wildcard expression to concrete indices that are open, closed or both.</summary>
 		public ExpandWildcards? ExpandWildcards { get => Q<ExpandWildcards?>("expand_wildcards"); set => Q("expand_wildcards", value); }
-		///<summary>TODO: ?</summary>
```

Comment:
Needs to be patched back in, in order to not break binary compatibility

Reply:
:+1: